data/campaigns/The_South_Guard/scenarios/03_A_Desparate_Errand.cfg

```diff
@@ -392,54 +412,109 @@
 
         [message]
             speaker=Ethiliel
-            message= _ "Indeed you are brave to face such foes. But I fear that alone you will fail and fall. One elf now lives who remembers the secrets of the undead that we learned when Haldric I came to our land. His name is Mebrin, and he lives, secluded in the hills, near here. I will take you to him."
+            message= _ "My answer will depend on your motives, young human. What would you do, knowing the secrets of the unholy ones?"
+        [/message]
+
+        [message]
+            speaker=Deoran
+            message= _ "I would lead my men south of the river to find and destroy the source of these beings of darkness."
+        [/message]
+
+        [message]
+            speaker=Vardanos
+            message= _ "A fanciful delusion! A human untrained in the mystic arts stands no chance against the undead, especially not one so young!"
+        [/message]
+
+        [message]
+            speaker=Deoran
+            message= _ "I have proven myself capable of defending my land and leading my men to victory—"
+        [/message]
+
+        [message]
+            speaker=Eltenmir
+            message= _ "In battles against other <i>humans</i>, young commander! To fight such a profane evil is another matter altogether."
+        [/message]
+
+        [message]
+            speaker=Deoran
+            message= _ "But surely I cannot simply do nothing either!"
+        [/message]
+
+        [message]
+            speaker=Minister Hylas
+            message= _ "<i>I warned him that the elves would not easily agree to help us.</i>"
+        [/message]
+
+        [message]
+            speaker=Sir Gerrick
+            message= _ "<i>Indeed, but he is right as well. We must do something, or our people will surely be slaughtered.</i>"
+        [/message]
+
+        [message]
+            speaker=Ethiliel
+            message= _ "Enough, all of you. Human, you are indeed brave to face such foes, but I fear that you will fall to the darkness if you choose to fight it alone. There is one elf who still remembers the secrets of the undead that we learned when Haldric I came to our land. His name is Mebrin, and he lives secluded in the hills south of here."
+        [/message]
+
+        [message]
+            speaker=Sidaurios
+            message= _ "Ethiliel, surely you are not-"
+        [/message]
+
+        [message]
+            speaker=Ethiliel
+            message= _ "<i>I</i> will accompany the human and his men to Mebrin. I will not ask you to place yourselves in danger for these humans, but you will not presume to tell me what to do. Is that fair?"
+        [/message]
+
+        [message]
+            speaker=Sidaurios
+            message= _ "Yes, very well."
         [/message]
 
         [message]
             speaker=Deoran
             image=portraits/deoran-glad.png
-            message= _ "My thanks, lady."
+            message= _ "Many thanks for your aid, my lady."
```

Comment:
The original line is stilted and the new line an improvement, but no aid has been received yet. Maybe "I welcome your aid, my lady."